docs/conf/portland/2018/news/announcing-speakers.rst

```diff
@@ -0,0 +1,60 @@
+* Havi Hoffman – `Not the docs: Shepherding a developer blog of many voices across many timezones <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-havi-hoffman>`_
+* Jen Lambourne – `Research like you’re wrong: Lessons from user research gone rogue <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-jen-lambourne>`_
+* Kat King – `Building Empathy-Driven Developer Documentation <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-kat-king>`_
+* Larry Ullman – `Making Your Code Examples Shine <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-larry-ullman>`_
+* Neal Kaplan – `Where do I start? The art and practice of documentation triage <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-neal-kaplan>`_
+* Sarah Day – `Starting from Scratch: Finding and Hiring Junior Writers <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-sarah-day>`_
+* Steve Stegelin – `Graphic Content Warning: The Pros, Cons, and Alternatives to Screenshots <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-steve-stegelin>`_
+* Ted Hudek – `Eight Essential Tips for the Enlightened Tech Writer <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-ted-hudek>`_
+* Thursday Bram – `What They Don't Tell You About Creating New Style Guides <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-thursday-bram>`_
+
+You can head over to the `speaker page <http://www.writethedocs.org/conf/portland/2018/speakers/>`_ to see the full abstracts. We hope there’s something there for everyone to enjoy, and also something that will stretch your horizons a little bit, too.
+
+
+Tickets are going fast!
+-----------------------
+If you don't have your ticket yet, now's a great time to snap one up. The conference has sold out for the last several years, so if you're planning to come, don't leave it to the last minute.
+
+`Buy your tickets here! <http://www.writethedocs.org/conf/portland/2018/tickets/>`_
+
+Planning your trip
+------------------
+In the next few weeks, we’ll be announcing more details of the conference events: the full schedule, the annual hike on Saturday afternoon, the writing day and reception on Sunday, and the official party on Monday evening. But on top of that, there's ALL SORTS of stuff to do in Portland. 
+
+We've got a couple of excellent volunteers – Mo Nishiyama and Alicia Duncan – who are heading up our Explore Portland committee. They're already hard at work putting together resources to help you plan your visit! Check out the `Visit Portland <http://www.writethedocs.org/conf/portland/2018/visiting/>`_ page on the conference website, or pop into the `#wtd-conferences <https://writethedocs.slack.com/messages/wtd-conferences>`_ channel on Slack to ask a question. 
```

Comment:
link to volunteers' twitter accounts?

Reply:
I don't have em handy at the moment, but I can add em in later.